pkg/client_data/lib/page_data.dart

```diff
@@ -2,11 +2,16 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+import 'dart:convert';
+
 import 'package:json_annotation/json_annotation.dart';
 import 'package:meta/meta.dart';
 
 part 'page_data.g.dart';
 
+/// Codec to serialize the [PageData] JSON content on the HTML page.
+final pageDataJsonCodec = json.fuse(utf8).fuse(base64);
```

Comment:
There is strictly no need for base64 encoding, but I'm okay with this :)

Reply:
The non-base64 JSON string was longer, as all the `"` got escaped to `&quot;`, and it was ugly to look at it. This way, it is just a bunch of bytes :)